src/util/contribution-types.js

```diff
@@ -74,6 +74,10 @@ const defaultTypes = function(repoType) {
       symbol: '🚧',
       description: 'Maintenance',
     },
+    mentoring: {
+      symbol: '🧑‍🏫',
```

Comment:
It's bizarre that the emoji shows up as the 2 emojis when on the issue it was different (viewing from the same browser, OS and device).

Reply:
Yes had the same thing on the app yesterday and in PyCharm in the markdown file but not the js file

Reply:
Also, is there a reason behind adding a space after `function` which is not done anywhere in the codebase (as far as arrow functions go)?

Reply:
Yes I also saw that, did do that on purpose, must have been an auto formatting thing, I'll remove them again

Reply:
they seem to get added by the pre-commit tasks? I remove the white spaces, I see them in `git diff` but when I run `git add` and `git commit` it tells me that my commit is empty ... I've changed on GitHub now

Reply:
Oh okay.